framework/src/executioners/FEProblemSolve.C

```diff
@@ -68,10 +68,18 @@ FEProblemSolve::validParams()
   params.addParam<Real>("l_abs_tol", 1.0e-50, "Linear Absolute Tolerance");
   params.addParam<unsigned int>("l_max_its", 10000, "Max Linear Iterations");
   params.addParam<unsigned int>("nl_max_its", 50, "Max Nonlinear Iterations");
+  params.addParam<unsigned int>("nl_forced_its", 0, "The Number of Forced Nonlinear Iterations");
```

Comment:
any reason you preferred 'forced' over 'min'?

Reply:
Just because we always mentioned forcing N iterations. Either names are ok for me

Reply:
We use "forced" here since PETSc is using the similar thing. I simply want to keep consistent with PETSc. https://www.mcs.anl.gov/petsc/petsc-current/docs/manualpages/SNES/SNESSetForceIteration.html

We are implementing MOOSE version of this capability  (a better one). PETSc does one force iteration only, while in moose we can do an arbitrary number of iterations 

I agree that either name will work. Name is always  the hardest part of coding, lol.

Reply:
Should we use "force" instead of "forced"? PETSc has an option like:  `-snes_force_iteration`

Reply:
Have no preferences between force and forced. Semantically the same.